cypress/support/cypress-support/src/globalSetter.js

```diff
@@ -0,0 +1,31 @@
+const UTILS = require('./utils');
+const CONSTANTS = require('../../constants/constants');
+
+global.setterSuccess = (message = 'successful response') => {
+  const methodName = UTILS.getEnvVariable(CONSTANTS.REQUEST_OVERRIDE_METHOD);
+  cy.log(`[${methodName}] Success: ${message}`);
+  return { status: 'success', message };
+};
+
+global.setterFailure = (message, error) => {
+  const methodName = UTILS.getEnvVariable(CONSTANTS.REQUEST_OVERRIDE_METHOD);
+  const errorMessage = `${methodName} ${message || `FCS setter for fcsSetter.${methodName} failed`}`;
```

Comment:
It looks like the formatting is different than for success? Let's have the messaging be consistent:

"[Setter Method (function)] (message)"

Ex: "[Setter Method fcs.setClosedCaptions] FCS Setter failed/succeeded"

Reply:
Addressed